src/app/games/tablut/tests/TablutMove.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { TablutRules } from '../TablutRules';
+import { TablutMinimax } from '../TablutMinimax';
+import { TablutMove } from '../TablutMove';
+import { TablutState } from '../TablutState';
+import { Coord } from 'src/app/jscaip/Coord';
+import { NumberEncoderTestUtils } from 'src/app/jscaip/tests/Encoder.spec';
+import { RulesFailure } from 'src/app/jscaip/RulesFailure';
+
+describe('TablutMove', () => {
+
+    it('encoder should be correct', () => {
+        const rules: TablutRules = new TablutRules(TablutState);
+        const minimax: TablutMinimax = new TablutMinimax(rules, 'TablutMinimax');
+        const firstTurnMoves: TablutMove[] = minimax.getListMoves(rules.node);
+        for (const move of firstTurnMoves) {
+            NumberEncoderTestUtils.expectToBeCorrect(TablutMove.encoder, move);
+        }
+    });
+    it('TablutMove creation, as a MoveCoordToCoord, should throw when created immobile', () => {
```

Comment:
peut-être mettre un describe('creation') { it('should throw ...') plutôt que TablutMove > TablutMove creation should ...

Reply:
dead folder :D